instrumentation/opentelemetry-instrumentation-threading/src/opentelemetry/instrumentation/threading/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#To-do : what is the actual version of threading?
+_instruments = ("threading >= 1.0, < 3.0",)
```

Comment:
@srikanthccv Since threading is in-built in python, where do I find the versions?

Reply:
No, you will not have any version for this. This should be just empty.